config/development.toml

```diff
@@ -229,7 +229,7 @@ stripe = { long_lived_token = false, payment_method = "wallet", payment_method_t
 checkout = { long_lived_token = false, payment_method = "wallet"}
 
 [connector_customer]
-connector_list = "bluesnap, stripe"
+connector_list = "bluesnap,stripe"
```

Comment:
It's quite easy to make the same mistake of including spaces again. Can we have this configuration variable as a `Vec<String>` instead?

If you do make that change, ensure you also include it in the list of keys to be parsed as a list:

https://github.com/juspay/hyperswitch/blob/bc4ac529aa981150de6882d425bd274bc6272e30/crates/router/src/configs/settings.rs#L512-L513

Reply:
Yes, right. There are other values too which we are reading as string and then `deserializing` it to a `Vec<String>`.  We have to find a better way to do this. Linking #1109 as this refactor can be taken up in this issue